boards/posix/doc/arch_soc.rst

```diff
@@ -169,6 +169,23 @@ In the previous example, modifying the code as follows would work:
     #endif
    }
 
+.. _posix_arch_unsupported:
+
+Significant unsupported features
+********************************
+
+Currently, these are the most significant features which are not supported in this architecture:
+
+* :ref:`User mode/userspace <usermode_api>`: When building for these targets,
+  :kconfig:option:`CONFIG_USERSPACE` will always be disabled,
+  and all calls into the kernel will be done as normal calls.
+
+* Stack checks: :kconfig:option:`CONFIG_HW_STACK_PROTECTION`,
+  :kconfig:option:`CONFIG_STACK_CANARIES`, and
+  :kconfig:option:`CONFIG_THREAD_ANALYZER`.
+  Due to Zephyr allocated threads' stacks not being used as in other architectures. Check
```

Comment:
Better yes! It just feels odd that the sentence has no verb. LMK if you feel the below is better.

```suggestion
  This is due to how Zephyr allocated threads' stacks are not `actually` being used like they are in other architectures. Check
```

Reply:
Changed :)